katas/langtonant/Giovanni/langtonsant.test.js

```diff
@@ -0,0 +1,38 @@
+import { turn } from './langtonsant';
+//import { advance } from './langtonsant';
+import { ant } from './langtonsant';
+
+test('Function that turns the ant according to the color of its position', () => {
```

Comment:
We need to add a test for final functionality, given a number of steps, validate the final status of the grid